File: AbstractFactoryDesignPattern/NotificationService_Example/Message/EmailMessage.java
package AbstractFactoryDesignPattern.NotificationService_Example.Message;

public class EmailMessage implements Message {
	String toEmailID;
	String body;

	@Override
	public void setContent(String to, String body) {
		this.toEmailID = to;
		this.body = body;
	}

	@Override
	public String formatMessage() {
		return "Email to <" + this.toEmailID + ">: " + this.body;
	}
}

File: AbstractFactoryDesignPattern/NotificationService_Example_Python/Message/EmailMessage.py
from AbstractFactoryDesignPattern.NotificationService_Example_Python.Message.NotificationMessage import NotificationMessage

class EmailMessage(NotificationMessage):

	def __init__(self) -> None:
		self.toEmailID : str = ""
		self.body : str = ""

	def setContent(self, to : str, body : str) -> None:
		self.toEmailID = to
		self.body = body

	def formatMessage(self) -> str:
		return "Email to <" + self.toEmailID + ">: " + self.body